spring-cloud-stream-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -492,4 +494,146 @@ Then do this:
 
 This would upload the docker image to the docker hub in your custom repository.
 
+=== General FAQ on Spring Cloud Stream App Starters
+
+In the following sections, you can find a brief faq on various things that we discussed above and a few other infrastructure related topics.
+
+
+. What are Spring Cloud Stream Application Starters? +
+Spring Cloud Stream App Starters are individual repositories hosted under the GitHub organization - spring-cloud-stream-app-starters - on github (http://github.com/spring-cloud-stream-app-starters) +
+Project page: http://cloud.spring.io/spring-cloud-stream-app-starters/
+
+
+. What is the parent for stream app starters? +
+The parent for all app starters is `app-starters-build` which is coming from the core project. https://github.com/spring-cloud-stream-app-starters/core
+
+. Why is there a BOM in the core proejct? +
+Core defines a BOM which contains all the dependency management for common artifacts. This BOM is named as `app-starters-core-dependencies`.
+ We need this bom during app generation to pull down all the core dependencies.
+
+. What are the contents of the core BOM? +
+In addition to the common artifacts in core, the app-starters-core-dependencies BOM also adds dependency management for spring-cloud-dependencies which will include spring-cloud-stream transitively.
+
+. Where is the core BOM used? +
+There are two places where the core BOM is used. It is used to provide compile time dependency management for all the starters.
+This is defined in the `app-starters-build` artfiact. This same BOM is referenced through the maven plugin configuration for the app generation.
+The generated apps thus will include this bom also in their pom.xml files.
+
+. What are the artifacts that the parent artifact (`app-starters-build`) include from the core bom to be part of all the starters by default and there by making them part of the generated apps? +
```

Comment:
It isn't clear what this question is about.  Is it about what specific spring cloud stream dependencies get pulled in transitively?